routes/index.js

```diff
@@ -12,12 +12,15 @@ import permissionRoute from "./permission/index.js";
 
 import employeeRoute from "./employee/index.js";
 
+import leaveRequest from "./leaveRequest/index.js";
+
 const protectedRouter = express.Router();
 const unProtectedRouter = express.Router();
 
 // Protected Routes
 
 // Un-Protected Routes
+unProtectedRouter.use("/leaveRequest", leaveRequest);
```

Comment:
Why leave route is unprotected?

Reply:
Aoa sir, so we're just following the boilerplate however if i've searched it seems to rouets those that need authorization are actually protected but sir in that all the most of the routes will get protected

Reply:
All the routes must be protected except login and signUp. @Azmat7860 @Sameerthe-Big-O @abubakar105

Reply:
Ok Sir